### DISCUSSION TITLE
"... observed superclusters are an order of magnitude larger than the theoretical ones predicted by cosmological ΛCDM simulations"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#28] [Astrophysics and Physics]
**"Watersheds of the Universe: Laniakea and five newcomers in the neighborhood"** A. Dupuy, H.M. Courtois, submitted to A&A (AA/2023/46802) [arXiv:2305.02339](https://arxiv.org/abs/2305.02339) (2023-5-3)

"This article delivers the dynamical cosmography of the Local Universe within $z = 0.1$.  Laniakea, our home supercluster's size is confirmed to be $2\times 10^6$ (Mpc $h^{-1})^3$. Five more superclusters are now dynamically revealed in the same way: Apus, Hercules, Lepus, Perseus-Pisces and Shapley. Also, the central repellers of the Bootes and Sculptor voids are found and the Dipole and Cold Spot repellers now appear as a single gigantic entity. Interestingly ***the observed superclusters are an order of magnitude larger than the theoretical ones predicted by cosmological ΛCDM simulations***."

**Comment by khuramonline** [comment]
Following is a short post. I have shown that official distance of Virgo Cluster is wrong.

https://www.quora.com/How-far-away-is-the-Virgo-Cluster/answer/Khuram-Rafique

Method is very simple. I have mentioned the (i) official angle of view and (ii) official diameter of Virgo. Then readers are invited to apply Trigonometry by themselves.

There are also few links to other short posts. Distances of Andromeda, and Coma Cluster are also wrong in this way.

However, following is the base link. It was for the first time (2019) when I pointed out that distances of all the far off galaxies are wrong by way of huge understatements. This is link to relevant section of my book (2019):

https://zenodo.org/record/3817879

**Comment by khuramonline** [comment]
Above was my first post. I am Khuram Rafique. The links in above post are not working so readers can manually open in browser. I am new here and may be yet have to figure out why external links point to internal 404 page.

**Reply by HanDeBruijn** [reply]
The links work fine for me, Khuram

Han

**Comment by khuramonline** [comment]


**Reply by ftessier** [reply]
That conversation is quite amazing! Thanks for sharing.

Large language models (LLM) AIs don't store any _facts_ internally; for example, nowhere in the neural net is the string "redshift" stored. That is why they are amazing. But this is also why we can't yet expect to attribute any "truth" value to their statements. For the moment they only get the language "right". The fact that they are so "intelligent" with _only that_ was a surprise for everyone (especially their designers) and IS the interesting question that reaches way back to the philosophy of language and thought, and the link between both.

In my experience, current LLMs are immensely useful when you know what you (and the AI) are talking about, or you can verify statements with an independent trustworthy source; and immensely useless if you don't or can't. I guess that is what makes them sound human in the first place 😄

**Comment by khuramonline** [comment]
All the official distances of galaxies are wrong. Redshift-distance was originally a linear relationship but later on it was contaminated by the SR based formulas that made it into non-linear relationship.

How actually it is implemented is something like... **Limit redshift approaching to infinity, distance approaching to 13.6 billion light years or may be 13.7 billion light years.**

There are learned persons at this forum who might be the authors of online Cosmology Calculators. Anyone can check from any online cosmology calculator that for any insane high value of redshift e.g. 2 million or whatever, the (light travel) distance shall remain 13.6 billion light years or so.

For a simple linear relationship of redshift and distance, the (light travel) distance for a galaxy having redshift 13 corresponds to the distance of 181 billion light years.

Not only redshifts, there is also independent proof that actual distances are at much higher scale than official distances. That proof is Gravitational Lensing.

**Yes, rather than the proof of the existence of "dark matter", Gravitational Lensing is actually a proof that official distances of galaxies are hugely understated.**

I have explained this thing in my second book (2019). Following is link to relevant section of book:

**[Gravitational Lensing Section - Book: "Philosophy Unscrambles Dark Matter" ](https://zenodo.org/record/3817872)**

**Reply by khuramonline** [reply]
Following is concluding para from above referred section:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/daa69d34-ebe6-437c-b704-fdd2246f5254)

So actually three independent lines of proofs indicate that official distances of galaxies are hugely understated. Those three independent proofs are:

**1. Simple Euclidean Geometry / Trigonometry,
2. Redshift-Distance linear relationship and;
3. Gravitational Lensing.**

**Reply by RedshiftDrift** [reply]
For redshifts up to z = 0.1 relevant to the article above, the difference between ΛCDM and the correct relationship is <4%, too small to be detectable with our current observations because of the proper motions of galaxies.

The linear distance-redshift is also incorrect because the redshift mechanism is a loss mechanism and produces an exponential dependence with distance.  Riess demonstrated this observationally by detecting what is [interpreted in ΛCDM as an acceleration of expansion](https://iopscience.iop.org/article/10.1086/300499/fulltext/), which is a deviation from the linear dependence.

In fact, galaxies at redshift $z = 13$ are at a distance of 34.8 Gly according to [this calculator](https://a-cosmology-group.github.io/acg/code/marmet_l/cosmology-calculator.html).

**Reply by khuramonline** [reply]
Since Gravitational Lensing cases are proper indicators of distances ... so I think actual distances can be worked out by way of reconciliation of linear redshift distance and lensing based distances. 

**Off course only thing needed is to abandon the idea that gravitational lensing cases are indicators of dark matter.**

It is possible that such reconciliation may give the value of distance of z=13 redshift galaxy something close to 34.8 Gly or may be close to 181 Gly or may be an inbetween value is obtained.

**Comment by khuramonline** [comment]
**"Dark Matter Ring is a strong evidence in support of Dark Matter"**

I have debunked this claim using same argument that **official distances of galaxies are hugely understated**. It is further explained how exactly same quantity of matter but with greater distance solves the problem of dark matter ring without invoking or requiring anything like dark matter. Following is the relevant section of book:

**[Dark Matter Ring and Dark Matter - Book: "Philosophy Unscrambles Dark Matter"](https://zenodo.org/record/3550313)**
